### PR TITLE
Include some information in the README about building the add-on and using runfx

### PR DESCRIPTION
**Pull request by ianb**

### CODE REVIEW

**Comment by mozilla-autolander-deprecated** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

File: README.md [hunk 1]
```diff
@@ -8,6 +8,31 @@ window.
...
+```shell
+$ ./bin/runfx.js -p nightly -p ./loop-dev/
+```
+
+**One time only**: This will use a new profile created in a `./loop-dev/`
+directory.  When Firefox opens you must go to the URL `about:config` and
+look for the setting `xpinstall.signatures.required`.  This must be set to
+**false**.  Then go to
+**Tools &gt; Add-ons** and find the Firefox Hello addon.  You must manually
+re-enable the add-on.
```
Comment (Standard8) [inline README.md:34]: nit: I think its reasonable just to say "enable".

**Comment by Standard8** [comment]
I think ideally I'd like this to be split out to a separate file, as we're going to be needing to add a lot more to it - for building, running, standalone, testing etc. However, if you want to land it as-is for now, and we'll move it later, I'm fine with that.

**Comment by dmose** [comment]
Shouldn't that be '-b nightly -p ...'?

**Comment by mozilla-autolander-deprecated** [comment]
Autolander could not find a bug number in your pull request title. All pull requests should be in the format of: Bug [number] - [description].

**Comment by ianb** [comment]
I'll infer @Standard8's response as an r+ (with the -b fix)

**Pull request merged**